lib/API.js

```diff
@@ -836,141 +836,151 @@ API.prototype._startJson = function(file, opts, action, pipe, cb) {
   if (typeof(cb) === 'undefined' && typeof(pipe) === 'function')
     cb = pipe;
 
-  if (typeof(file) === 'object')
-    config = file;
-  else if (pipe == 'pipe')
-    config = Common.parseConfig(file, 'pipe');
-  else {
-    var data = null;
-
-    var file_path = path.join(that.cwd, file);
+  getConfig().then(function (config) {
```

Comment:
Need to use `reject()` and catch it here to call `cb` or exit the CLI with the error.